models/order.js

```diff
@@ -0,0 +1,47 @@
+const { required } = require('joi')
+const mongoose = require('mongoose')
+
+const OrderSchema = new mongoose.Schema({
+    item: {
```

Comment:
I think you want this items array to reference the item schema you have defined in item.js  Try looking at the [subdocuments section](https://mongoosejs.com/docs/subdocs.html) so they reference each other

Reply:
Mr Zach, thanks for reviewing my code.  Order schema does not reference the item schema. 
I am making an order app, that will contain the items that users will create orders and add the items in it. I think my logic is correct. Just I need a little bit to improve my code.